src/test/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodecRegistryTest.java

```diff
@@ -81,9 +82,29 @@ public void codecToCQL(DataType cqlType, Object fromValue, Object expectedCqlVal
   }
 
   private static Stream<Arguments> codecToCQLTestCases() {
+    // Arguments: (CQL-type, from-caller, bound-by-driver-for-cql
+    // Note: all Numeric types accept 3 Java types: Long, BigInteger, BigDecimal
     return Stream.of(
-        Arguments.of(DataTypes.INT, BigDecimal.valueOf(100), 100) // second 100 is an int
-        );
+        // Integer types:
+        Arguments.of(DataTypes.BIGINT, -456L, -456L),
```

Comment:
nit: consider rename to something like validCodecToCQL.... now you have out of range below.

Reply:
Excellent point, changing